src/angular/loader/loader.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, ViewContainerRef, Inject, OnInit, OnDestroy } from "@angular/core";
+import template from "./loader.component.html";
+import { LoaderService } from "./loader.service";
+
+export enum LoaderSize {
+    large = 'large',
+    medium = 'medium',
+    small = 'small',
+}
+
+@Component({
+    selector: "sdc-loader",
+    template: template
+})
+
+export class LoaderComponent implements OnInit, OnDestroy {
+    @Input() active: number;
```

Comment:
1. This is not mandatory input.
2. Add Output for `active` to make it two-way binding. (`@Input active` and `@Output activeChange`). Then it would be possible to use this attribute as `[(active)]="myActiveVar"`.